addons/main_a3/config.cpp

```diff
@@ -19,7 +19,7 @@ class CfgSettings {
         class Versioning {
             class PREFIX {
                 class Dependencies {
-                    CBA[] = {"cba_main", { 1,0,0 },"(true)"};
+                    CBA[] = {"cba_main", { 1,0,0},"(true)"};
```

Comment:
Ugh.... how about `{1, 0, 0}` to make it less hurtful on the eyes?

Reply:
Or `{1,0,0}` so there are no spaces anywhere.